.circleci/config.yml

```diff
@@ -12,7 +12,8 @@ jobs:
           key: dependency-cache-{{ checksum "package.json" }}
       - node/install:
           install-yarn: true
-          node-version: latest
+          node-version: 15.7.0
```

Comment:
should we add these lines to the deploy-rc task too?

Reply:
Yeah I think you're right.  Added now.